### DISCUSSION TITLE
Visningsfejl på mobil

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#19] [Bugs] [answered]
Når man opretter en ny bruger på en mobiltelefon (undtaget mobiler med meget store skærme - større end iphone 11), så forhindrer visningen lige nu, at man kan ændre sin fødselsdato.

Vi tror, at de fleste brugere opretter sig via mobiltelefon, hvilket også forklarer, at vi lige nu går mange brugere ind, som er 100 år.

![IMG_5325 (1)](https://github.com/user-attachments/assets/37620f54-7771-4fd4-8d44-0e813d4bd4df)

Oprettelsespopuppen ved brugeroprettelse på mobil (de fleste almindelige skærmstørrelser) skal tilrettes, så man som minimum kan ændre sin alder.

**Comment by mikkelsl** [comment, marked as answer]
Løses her: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=73828517
